src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -287,6 +288,51 @@ public void rename(String newName) {
     }
 
     /**
+     * Checks if a new version of the file can be successfully uploaded by using the preflight check.
+     * @param  name        the name to give the uploaded file.
+     * @param  fileSize    the size of the file used for account capacity calculations.
+     * @param  id          the id of the file that has a new version being uploaded.
+     */
+    public void preflightUploadCheck(String name, long fileSize, String id) {
```

Comment:
I'd prefer that this method be added to BoxFile instead so that an ID doesn't need to be passed in.